test_runner/performance/test_parallel_copy_to.py

```diff
@@ -6,7 +6,7 @@
 
 
 async def repeat_bytes(buf, repetitions: int):
-    for i in range(repetitions):
+    for _i in range(repetitions):
```

Comment:
Can it be replaced with 

```python
yield from itertools.repeat(buf, repetitions)
```

???

Reply:
Another `ruff` autofix. but perhaps in this case we should.